test/Table_Tests/src/Database/Postgres_Spec.enso

```diff
@@ -100,35 +100,37 @@ postgres_specific_spec connection db_name setup =
         connection.execute_update 'DROP TABLE "'+tinfo+'";'
 
     tinfo = Name_Generator.random_name "Tinfo"
-    connection.execute_update 'CREATE TEMPORARY TABLE "'+tinfo+'" ("strs" VARCHAR, "ints" INTEGER, "bools" BOOLEAN, "reals" REAL, "doubles" DOUBLE PRECISION)'
+    connection.execute_update 'CREATE TEMPORARY TABLE "'+tinfo+'" ("strs" VARCHAR, "ints" BIGINT, "bools" BOOLEAN, "doubles" DOUBLE PRECISION)'
```

Comment:
Why have we removed the bit checking on these?

Reply:
I'm now using `update_database_table`. This is checking if column types fit.

But then, the _source_ in-memory table only supports 64-bit integers and floats. Which are too big and so they do not fit the target, so the update opperation errors with `Column_Type_Mismatch`. To make this work, we'd first need to cast the source table into `Integer 32-bit`, but that is not possible as #5159 is still not implemented.

Reply:
Ideally, we should have the possibility. But also these tests are not most important so I decided it is fine to amend them